### DISCUSSION TITLE
Deadline Reminder!!!

### DISCUSSION THREAD

**Discussion by sarthakpati** [FETS-AI/Challenge#73] [General]
Hello Participants,

Thank you for your interest in the first ever computational challenge in Federated Learning!!

This is a reminder that the **deadline** for initiating your paper submission on the [CMT system](https://cmt3.research.microsoft.com/BrainLes2021) is the **19th of July**! You can then finalize your submission by the **26th of July**. Please **_note_** that the initial submission only requires to indicate the task(s) you intend to participate.

As always, please follow us [on Twitter](https://twitter.com/FeTS_Challenge) and post here for detailed responses!

Kind regards,
FeTS Challenge Organizing Committee

**Comment by sarthakpati** [comment]
**UPDATE ON EXTENSION (2021/07/15):**

Please note that the deadline for notebook submission is **still** the **26th of July** at **2359 EST**. However, the manuscript can now be submitted by **2nd of August** at **2359 EST**. Additionally, changes to the **parameters** used in the notebook (**not** any algorithmic code) can also be done till the **2nd of August** at **2359 EST**.

Reach out to us on email/Twitter for further clarification!

Kind regards,
FeTS Challenge Organizing Committee

**Reply by Vishruth-Sham** [reply]
Hi @sarthakpati . By parameters, do you mean numbers in our code, or epochs,learning rate etc.
If I have a threshold value, can I change it? 

For example, I have this variable "x"
if x>fl_round/2: 
      do something.

Can I change 2 to 3?

**Reply by sarthakpati** [reply]
> do you mean numbers in our code, or epochs,learning rate etc.

Yes

> If I have a threshold value, can I change it?

Yes.

All the best.

**Reply by Vishruth-Sham** [reply]
Thanks! That's vey helpful

**Reply by sarthakpati** [reply]
You're welcome. All the best. 😄 👍🏽
